### ISSUE TITLE
Lab needs old version of Elixir.

### ISSUE THREAD

**Issue by kenokenobingo** [starbelly/lab#3]
During compilation: `warning: the dependency :lab requires Elixir "~> 1.5.1" but you are running on v1.6.5`.

### My environment
* Lab 0.1.6
* version of Gitlab

**Comment by starbelly** [comment]
@kenokenobingo This should not be a problem... I suppose it's also time to make this lib truly useful as well :) Hang tight.

**Comment by kenokenobingo** [comment]
@starbelly Indeed, it’s no problem. I just wanted to make it public. Looking forward to the truly useful lib! :)
